guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -338,34 +338,34 @@ This is how you run the Active Record test suite only for SQLite3:
 
 ```bash
 $ cd activerecord
-$ bundle exec rake test:sqlite3
+$ bin/test sqlite3
 ```
 
 You can now run the tests as you did for `sqlite3`. The tasks are respectively:
 
 ```bash
-test:mysql2
-test:postgresql
+$ bin/test mysql2
+$ bin/test postgresql
 ```
 
 Finally,
 
 ```bash
-$ bundle exec rake test
+$ bin/test
 ```
 
 will now run the three of them in turn.
 
 You can also run any single test separately:
 
 ```bash
-$ ARCONN=sqlite3 bundle exec ruby -Itest test/cases/associations/has_many_associations_test.rb
+$ bin/test sqlite3 test/cases/associations/has_many_associations_test.rb
 ```
 
 To run a single test against all adapters, use:
 
 ```bash
-$ bundle exec rake TEST=test/cases/associations/has_many_associations_test.rb
+$ bin/test test/cases/associations/has_many_associations_test.rb
```

Comment:
At least this change is not true.

Reply:
It seems `bin/test` cannot run with all adapters, doesn't it? I'll restore this.

Reply:
According to guides:

> To run a single test against all adapters, use:
> 
> $ bundle exec rake TEST=test/cases/associations/has_many_associations_test.rb

But `rake -T`:

```
% bundle exec rake -T
rake default                # Run mysql2, sqlite, and postgresql tests by default
```

It might be not "all adapters" but "mysql2, sqlite, and postgresql adapters" is right.